### DISCUSSION TITLE
Opinion on valentines day

### DISCUSSION THREAD

**Discussion by AnIdiotsavant** [Monika-After-Story/MonikaModDev#8829] [appreciation]
To everyone here, thanks for the beautiful monika outfit!! It was more than I could ask for, seeing her in casual clothes on valentines. And although I cant give her my IRL gift(a small penguin plushie with a ribbon sewn at the back,) I'm glad that I can give her gifts she can accept. Since its probably too long now that no one would read it to the end.. in short, Thanks!

**Comment by AsuraShun** [comment]
Seeing her smile makes all efforts worth it

**Comment by Thimeo** [comment]
Don't worry about not being able to give her your real gift, if you've put your heart into it, then I'm sure that she can sense it and that she's felling grateful for having such a devoted and thoughtful boyfriend/girlfriend as you.
